### ISSUE TITLE
how do I clear gw2launcher cache

### ISSUE THREAD

**Issue by ParrotFaxe** [Healix/Gw2Launcher#392]
The problem I have is that I did a clean install of Win11 and imported the back-up file. However, this doesn't import the passwords properly since I didnt export them right. I also have an xml file with all the right data. 

But the problem is that when I clear all the accounts, the freshly loaded xml accounts start at ID 127 but I need them to start at ID 1, I want to clear all cache file so I have a "freshly" installed gw2launcher

**Comment by Healix** [comment]
To reset the ID, delete all of the accounts, then restart Gw2Launcher. Alternatively delete the %appdata%\Gw2Launcher folder to reset everything.

**Comment by ParrotFaxe** [comment]
Perfect and thank you for the Quick answer

**Comment by ParrotFaxe** [comment]
Can I also all point them towards thr same local.day file?

**Comment by Healix** [comment]
Yes, every account could use the same Local.dat. Keep in mind Local.dat is where GW2 stores your "local" settings (remembered login, sound, some graphics, trading post favorites, daily sorting).
